python_scripts/cross_validation_ex_05.py

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
I think that we need a bit of text in the beginning of the file to describe what this is about

Reply:
Yes I am working on the example right now. more to come ;)